### ISSUE TITLE
Setting trainable model parameters

### ISSUE THREAD

**Issue by jdegange** [ThilinaRajapakse/simpletransformers#279]
Hi - I am trying to control embedding and classification head layers within the models as trainable or frozen.  Generally in PyTorch I do this with something like:

`for param in model.parameters():
    param.requires_grad = False`

However, I am only able to do that through simpletransformers by accessing and changing corresponding _model.py scripts for a given task (NER, Classification), etc. 

Is there a more convenient way to control which weights we update gradients?

**Comment by ThilinaRajapakse** [comment]
This is not supported directly right now as Transformer models are generally fine-tuned for all layers.

**Comment by jdegange** [comment]
Got it, thanks
